Makefile

```diff
@@ -25,11 +25,7 @@ clean:  ## clean all build, python, and testing files
 	find . -name '*.pyo' -exec rm -fr {} +
 	find . -name '*~' -exec rm -fr {} +
 	find . -name '__pycache__' -exec rm -fr {} +
-	rm -fr .tox/
 	rm -fr .coverage
 	rm -fr coverage.xml
 	rm -fr htmlcov/
 	rm -fr .pytest_cache
-
-build: ## run tox / run tests and lint
-	tox
```

Comment:
i just fully removed the tox.ini file... didn't seem like you were using it anywhere?

Reply:
I think the only reason I wasn't using it anywhere was simply because this repo is so out-of-date. In other repos, I have started to centralize everything to the toxfile. In GH Actions I simply call `tox -e py` But to be honest.

Reply:
could put back in the tox file if you like

Reply:
No worries. It works for now. I may swap back if I want it but for now totally fine.